ghost/admin/app/styles/patterns/global.css

```diff
@@ -629,11 +629,18 @@ input[type="image"] {
     position: relative;
 }
 
+
 .show {
+    display: block;
+}
```

Comment:
What is the purpose for keeping these non-scoped utility classes in Ember's stylesheet?